docs/reference/api/platform/Assert.md

```diff
@@ -1,11 +1,31 @@
 ## Assert
 
-[Add description here.]
+Mbed-OS provides a set of macros which evaluates an expression and prints an error message if the expression evaluates to false. There are two types of macros, one for evaluating the expression during runtime and other can be used at compile-time. These macros are defined in mbed_assert.h.
 
-### Assert class reference
+### Assert macros reference
 
-[Embed class reference here. If you've never done this, please ask your editor.]
+[![View code](https://www.mbed.com/embed/?type=library)](https://os.mbed.com/docs/v5.6/mbed-os-api-doxy/group__platform__Assert.html)
 
-### Assert example
+### Assert usage example
+
+MBED_ASSERT macro can be used for runtime evaluation of expressions. If the evaluation fails, an error message will be printed out to bash in the below format. Note that MBED_ASSERT macro is available in debug builds only. 
```

Comment:
I assume debug builds here means debug + develop profiles?

Reply:
Good question, I really thought its just debug builds. Let me confirm and update the docs.